### ISSUE TITLE
python sample code

### ISSUE THREAD

**Issue by nobyt** [google/mediapipe#1348]
In the mediapipe document,
https://google.github.io/mediapipe/solutions/pose.html

Original image is saved to "annotated_image*.png instead of 
annotated image.
```
-  cv2.imwrite('/tmp/annotated_image' + str(idx) + '.png', image)
+  cv2.imwrite('/tmp/annotated_image' + str(idx) + '.png', annotated_image)
```

**Comment by jiuqiant** [comment]
Thanks for reporting this issue. We will fix this in the next release.

**Comment by jiuqiant** [comment]
Fixed in v0.8.1
